ci/jenkins/test.sh

```diff
@@ -1147,8 +1151,12 @@ fi
 
 trap clean_antrea EXIT
 if [[ ${TESTCASE} =~ "windows" ]]; then
-    if [[ ${TESTCASE} =~ "containerd" ]]; then
+    if [[ ${TESTCASE} =~ "containerd-ovs" ]]; then
+        WINDOWS_YAML_SUFFIX="windows-containerd-ovs"
+    elif [[ ${TESTCASE} =~ "containerd" ]]; then
         WINDOWS_YAML_SUFFIX="windows-containerd"
+    fi
+    if [[ ${TESTCASE} =~ "containerd" ]]; then
```

Comment:
why this?

Reply:
because in both containerd and containerd-with-ovs case , we need to execute this block
build_and_deliver_antrea_windows_and_linux_containerd_images
        if [[ ${TESTCASE} =~ "e2e" ]]; then
            run_e2e_windows
        else
            run_conformance_windows_containerd
        fi